### DISCUSSION TITLE
Expected Release of v13

### DISCUSSION THREAD

**Discussion by kaaax0815** [discordjs/discord.js#5631] [Q&A] [answered]
When is the expected Release of v13?

**Comment by kyranet** [comment, marked as answer]
v13's expected release time is at least once all `semver-major` PRs (besides API v9 + Threads) have been merged.

You can check said PRs easily by using the [`is:pr is:open label:"semver: major"`](https://github.com/discordjs/discord.js/pulls?q=is%3Apr+is%3Aopen+label%3A%22semver%3A+major%22+) filter.

**Reply by kaaax0815** [reply]
Okay thanks. Because I wanted to know when the Slash Commands Functionality will be released. Because it says that it will be included in v13

**Reply by kyranet** [reply]
There's no fixed date, it'll be released (with slash commands functionality) when the aforementioned PRs are merged.

**Reply by kaaax0815** [reply]
I know there is no fixed date but is there an expected or realistic date or timespan?

**Reply by kyranet** [reply]
Not really, we don't follow a release schedule and pretty much only release when it's ready.

If you want my personal estimate, given the activity PRs are having, I'd say 2 weeks.

**Reply by kaaax0815** [reply]
I really appreciate your time to help me! Thanks very much
